### PR TITLE
https://unit.atlassian.net/browse/NP-45237

### PR DESCRIPTION
**Pull request by StigNorland**
mediatype cause trouble fix

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ static Builder builder() {
 
     public String doSearch(ResourceAwsClient queryClient) {
         return
-            this.getMediaType().equals(MediaType.CSV_UTF_8)
+            this.getMediaType().is(MediaType.CSV_UTF_8)
```
Comment (torarnet) [inline search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java:52]: nullsafe?

Comment (StigNorland) [reply search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java:52]: Nei, men getMediaType er alltid fyllt inn

Comment (StigNorland) [reply search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java:52]: Jeg snur uttrykket for ordens skyld

**Review by torarnet — APPROVED** [review]
Maybe write a unit test for different accept headers, otherwise OK

**Comment by StigNorland** [comment]
> Maybe write a unit test for different accept headers, otherwise OK

It is tested in the Handler which is in another subproject  `shouldReturnTextCsvWithGivenAcceptHeader`

**Pull request merged**